marketplace-service/src/test/java/com/axonivy/market/service/impl/GHAxonIvyProductRepoServiceImplTest.java

```diff
@@ -232,7 +232,14 @@ void testExtractReadMeFileFromContents() throws IOException {
 
       List<GHContent> contents = List.of(mockReadmeFile, mockImageFile);
 
+
+
+      IntroductionDataModel introductionDataModel = new IntroductionDataModel();
+      introductionDataModel.setDescription(new HashMap<>(Map.of("en","Test README content")));
```

Comment:
- why not using Language.EN.getValue() for key?
- remove 2 redundant lines above
- it's better if you change some hard-codes to const in this test